zml/pjrtx.zig

```diff
@@ -2,6 +2,7 @@ const builtin = @import("builtin");
 const std = @import("std");
 
 const mlir = @import("mlir");
```

Comment:
this import is not needed for `pjrtx.zig`

Reply:
Addressed in https://github.com/zml/zml/pull/35/commits/860c9526ea529bcc7fe2d29c73668d66134b868f